core/backend-itwin-client/src/imodelhub/AzureFileHandler.ts

```diff
@@ -19,8 +19,8 @@ import {
   UserCancelledError,
 } from "@bentley/itwin-client";
 import { BackendITwinClientLoggerCategory } from "../BackendITwinClientLoggerCategory";
-import { downloadFileAtomic } from "../downloadFileAtomic";
 import { AzCopy, InitEventArgs, ProgressEventArgs, StringEventArgs } from "../util/AzCopy";
+import { BlobDownloader, ProgressData } from "./BlobDownloader";
```

Comment:
can we get rid of AzCopy?

Reply:
I indented to do that incrementally as azcopy is configured by many application inside and outside company. I like instead the application to remove azcopy configuration first and switch to default downloader and then remove azcopy support. 

Agents will be first to remove azcopy.

I can add Log message to warn that azcopy support will be removed.